src/github/tests/mod.rs

```diff
@@ -116,6 +118,38 @@ fn team_dont_add_member_if_invitation_is_pending() {
     "###);
 }
 
+#[test]
+fn org_member_not_sync() {
+    let mut model = DataModel::default();
+    let user = model.create_user("sakura");
+    let user2 = model.create_user("hitori");
+    model.create_team(TeamData::new("team-1").gh_team("members-gh", &[user, user2]));
+    let gh = model.gh_model();
+
+    // let gh_members = gh.org_members("rust-lang").unwrap();
+
+    let gh_members = HashSet::from([user2]);
+
+    println!("User data {:?}", gh_members);
+    model
+        .get_team("team-1")
+        .remove_gh_member("members-gh", user);
+
+    let gh_org_diff = model.diff_teams_gh_org(gh, gh_members);
```

Comment:
This function should only receive `gh`, not any additional parameters. The logic here is not fully correct, as you're passing GH members, but the function should diff `team` (TOML) members. The list of member should be taken from the `GithubMock`.